live/Artillery/index.idx

```diff
@@ -1,3 +1,4 @@
+0.0.8 Support for the X3 and X4 series by porting settings from ArtillerySlicer V3.0.

```

Comment:
min_slicer_version is missing on the first line

add `min_slic3r_version = 2.6.0`